contrib/intro/index.rst

```diff
@@ -10,9 +12,26 @@ Introduction
 
 
 
+[Open question: how to divide content between this Introduction and the
+:ref:`home page <c_root>`?]
 
 Welcome!
 
+New to open source?
+===================
+
+Python is an open source project, with culture and techniques from the broader
+open source world.  You might find it helpful to read about open source in
+general.  A number of individuals from the Python community have contributed to
+a series of excellent guides at `Open Source Guides
+<https://opensource.guide/>`_.
+
+Anyone will find the following guides useful:
+
+* `How to Contribute to Open Source <https://opensource.guide/how-to-contribute/>`_
+* `Building Welcoming Communities <https://opensource.guide/building-community/>`_
+
+
 Healthy Collaboration
```

Comment:
Sentence case titles:
```suggestion
Healthy collaboration
```

And the others further down the page.

Reply:
Done.